importer/tests.py

```
@@ -1,3 +1,518 @@
+        mock_file = MagicMock()
+        mock_file.name = "test.txt"
+        mock_file.is_dir.return_value = False
+        mock_input_path.iterdir.return_value = [mock_file]
+
+        mock_path_class.return_value = mock_input_path
+        mock_path_class.home.return_value = mock_input_path
+
+        response = self.client.get("/api/directories/", follow=True)
+        data = json.loads(response.content)
+
+        self.assertIsNone(data["error"])
```

Ruff (0.15.0)

[warning] Use a regular `assert` instead of unittest-style `assertIsNone`

Replace `assertIsNone(...)` with `assert ...`

(PT009)